### ISSUE TITLE
"Interactive chart" is repeated on screen reader

### ISSUE THREAD

**Issue by timchoi1613** [highcharts/highcharts#20064]
There's an issue with the Highcharts HTML. When I focus on the chart area, the div element wrapping the SVG has a "tabindex=0" and the SVG has the aria-label "Interactive charts".

Setting tabindex without assigning a role causes accessibility issue. This results in having two nodes in the accessibility tree: the parent div (becomes a node due to tabindex) and the SVG. The SVG has a specific aria-label, and the parent div  also takes the same name from its content.

### Expected behaviour
Screen reader should not announce the ‘Interactive chart’ information twice when chart is focused

### Actual behaviour
Screenreader is announcing ‘Interactive chart’ information twice when chart is focused

### Live demo with steps to reproduce
![Screenshot 2023-11-02 at 4 17 11 PM](https://github.com/highcharts/highcharts/assets/131310052/590c0e44-604a-4294-a51c-ea8a708afc82)
![Screenshot 2023-11-02 at 4 21 15 PM](https://github.com/highcharts/highcharts/assets/131310052/1c174d2d-e579-4e67-bf30-af46bfe662a9)

The problem is fixed when I give a role to the div element that wraps the SVG element. For instance:
![Screenshot 2023-11-02 at 4 25 54 PM](https://github.com/highcharts/highcharts/assets/131310052/1e5091bf-f537-404b-9a85-74c7c99a3dc2)

#### Product version
8.1.2

#### Affected browser(s)
Chrome, Edge, Safari

**Comment by karolkolodziej** [comment]
Hi @timchoi1613!

Thank you for contacting us! I'm not sure if this is a bug, maybe @oysteinmoseng or @marvin19 could clarify.

In the meantime, you can change the following properties:
- [chartContainerLabel](https://api.highcharts.com/highcharts/lang.accessibility.chartContainerLabel)
- [svgContainerLabel](https://api.highcharts.com/highcharts/lang.accessibility.svgContainerLabel)

**Comment by timchoi1613** [comment]
@karolkolodziej Thanks for taking a look! Indeed, it's more of an a11y issue rather than a bug

**Comment by oysteinmoseng** [comment]
I agree that this is not a bug, but a question of good default values. We should test with various screen readers and log what is being announced, and tweak default values accordingly.

It may be possible that [svgContainerLabel](https://api.highcharts.com/highcharts/lang.accessibility.svgContainerLabel) should be empty by default.

**Comment by timchoi1613** [comment]
svgContainerLabel works. Thank you! But just like @oysteinmoseng said, it'd be great for HighChart to have empty by default.

**Comment by karolkolodziej** [comment]
So let's keep this issue open then 😉

**Comment by vsanse** [comment]
but how to use these properties when there are more than one chart on page.. as these properties will set aria-label as global option for all charts

**Comment by oysteinmoseng** [comment]
Hi @vsanse, although not documented yet in the official API, you can use a top level `lang` option per chart to set these options. See https://jsfiddle.net/3r8kd4qf/

**Comment by adityaraute** [comment]
> So let's keep this issue open then 😉

Why has this been reopened? 
Are you still looking for fixes here?

**Comment by karolkolodziej** [comment]
It was reopened because the `svgContainerLabel` could be set as an empty string as per @oysteinmoseng suggestions. 
It's here for grasp if you want to give it a go

**Comment by adityaraute** [comment]
> It's here for grasp if you want to give it a go

Will give it a try

**Comment by oysteinmoseng** [comment]
Apologies for the confusion I may have caused above, but it is not at all clear that this should be empty by default. Testing is required, to verify what is most intuitive setting for users across different screen reader & browser combos.
